src/main.tsx

```diff
@@ -1,16 +1,26 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 
-import App from "./App.tsx";
+import AppRoot from "./App.tsx";
 import "./index.css";
 
-const root = document.getElementById("root");
-if (root !== null) {
+const rootElement = document.getElementById("root");
+if (rootElement) {
+  ReactDOM.createRoot(rootElement).render(
+    <React.StrictMode>
+      <AppRoot /> {/* Updated component reference */}
+    </React.StrictMode>,
+  );
+} else {
+  console.error("root element is missing");
+}
+
+/*if (root !== null) {
```

Comment:
The commented-out code can be removed now.

Reply:
Removed the commented out code